pkg/pillar/scripts/device-steps.sh

```diff
@@ -503,6 +503,7 @@ if [ $MEASURE = 1 ]; then
 fi
 
 echo "$(date -Ins -u) Done starting EVE version: $(cat /run/eve-release)"
+/sbin/periodic_sigusr1.sh &
```

Comment:
I don't know how much the sigusr1 every 15 seconds will slow things down and also bloat the logs.

Can't we ssh in and manually kick off the script when we want the stack traces?

Reply:
Agreed with @eriknordmark here -- this script seems like such a debug tool that I can't see us adding it to the default code path.